src/main/scala/com/redis/sentinel/SentinelMonitor.scala

```diff
@@ -60,12 +63,12 @@ class SentinelMonitor (address: SentinelAddress, listener: SentinelListener, con
 
     sentinel = new SentinelClientPool(address)
     if (config.heartBeatEnabled) {
-      heartBeater = new SentinelHeartBeater {
+      heartBeater = Some(new SentinelHeartBeater {
         val sentinelClient: SentinelClient = new SentinelClient(address)
         def heartBeatListener: SentinelListener = listener
         def heartBeatInterval: Int = config.heartBeatInterval
-      }
-      new Thread(heartBeater).start()
+      })
+      heartBeater.map(new Thread(_).start())
```

Comment:
tom pushed everyone to use foreach instead of map :)

https://github.com/spaceapegames/scala-coding-guidelines